conanfile.py

```diff
@@ -14,7 +14,7 @@ class TaraxaConan(ConanFile):
 
     def requirements(self):
         self.requires("boost/1.76.0")
-        self.requires("cppcheck/2.3")
+        # self.requires("cppcheck/2.3")
```

Comment:
why this is disabled?

Reply:
added by accident during changes moving from another branch